src/docs/04_solution_strategy.adoc

```diff
@@ -1,24 +1,29 @@
 [[section-solution-strategy]]
-== Solution Strategy
+= Solution Strategy
 
+=== Technology Decisions
 
-[role="arc42help"]
-****
-.Contents
-A short summary and explanation of the fundamental decisions and solution strategies, that shape the system's architecture. These include
+==== Backend
 
-* technology decisions
-* decisions about the top-level decomposition of the system, e.g. usage of an architectural pattern or design pattern
-* decisions on how to achieve key quality goals
-* relevant organizational decisions, e.g. selecting a development process or delegating certain tasks to third parties.
+* Solid-auth-Client: users will in the end use this library developed by SOLID, which is open source, and free to use.
+
+* JSON: We will be using JSON to represent the routes, because we are familiar with it.
+
+*  solid-file-client: This a library from github that allow the interchange of files with the POD. It will be used to send and recieve the jsons and multimedia content.
+
+* GeoJson: It will be used to send the information to the map, as an abstraction layer in case there is a need to change the map.
+
+==== Frontend
+
+* Google Maps: We will need a map to show the users the route, we are selecting Google maps because os its simplicity and variaty of entry formats.
```

Comment:
Please check your sentence formulation.
... Google maps because os its ... variaty
... Google Maps because of its ... variety

Also some punctuation marks are missused.